test/jdk/sun/util/calendar/zi/GenDoc.java

```diff
@@ -154,24 +154,26 @@ int processZoneinfo(Timezone tz) {
             outD.mkdirs();
 
             /* If mapfile is available, add a link to the appropriate map */
-            if ((mapList == null) && (Main.getMapFile() != null)) {
-                FileReader fr = new FileReader(Main.getMapFile());
-                BufferedReader in = new BufferedReader(fr);
-                mapList = new HashMap<String,LatitudeAndLongitude>();
-                String line;
-                while ((line = in.readLine()) != null) {
-                    // skip blank and comment lines
-                    if (line.length() == 0 || line.charAt(0) == '#') {
-                        continue;
+            if (mapList == null) {
+                mapList = new HashMap<String, LatitudeAndLongitude>();
+                if (Main.getMapFile() != null) {
```

Comment:
You could retain the map file and use it in the following line

Reply:
Correct.  Thanks for the catch.